langchain4j-bedrock/src/main/java/dev/langchain4j/model/bedrock/internal/AbstractSharedBedrockChatModel.java

```diff
@@ -0,0 +1,110 @@
+package dev.langchain4j.model.bedrock.internal;
+
+import dev.langchain4j.data.message.ChatMessage;
+import dev.langchain4j.data.message.ChatMessageType;
+import dev.langchain4j.internal.Json;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider;
+import software.amazon.awssdk.auth.credentials.DefaultCredentialsProvider;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.bedrockruntime.BedrockRuntimeClient;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static java.util.stream.Collectors.joining;
+
+@Getter
+@SuperBuilder
+public abstract class AbstractSharedBedrockChatModel {
+    protected static final String HUMAN_PROMPT = "Human:";
+    protected static final String ASSISTANT_PROMPT = "Assistant:";
+    protected static final String DEFAULT_ANTHROPIC_VERSION = "bedrock-2023-05-31";
+
+    @Builder.Default
+    protected final String humanPrompt = HUMAN_PROMPT;
+    @Builder.Default
+    protected final String assistantPrompt = ASSISTANT_PROMPT;
+    @Builder.Default
+    protected final Integer maxRetries = 5;
+    @Builder.Default
+    protected final Region region = Region.US_EAST_1;
+    @Builder.Default
+    protected final AwsCredentialsProvider credentialsProvider = DefaultCredentialsProvider.builder().build();
+    @Builder.Default
+    protected final int maxTokens = 300;
+    @Builder.Default
+    protected final float temperature = 1;
+    @Builder.Default
+    protected final float topP = 0.999f;
+    @Builder.Default
+    protected final String[] stopSequences = new String[]{};
+    @Builder.Default
+    protected final int topK = 250;
+    @Builder.Default
+    protected final String anthropicVersion = DEFAULT_ANTHROPIC_VERSION;
+
+
+    /**
+     * Convert chat message to string
+     *
+     * @param message chat message
+     * @return string
+     */
+    protected String chatMessageToString(ChatMessage message) {
+        switch (message.type()) {
+            case SYSTEM:
+                return message.text();
+            case USER:
+                return humanPrompt + " " + message.text();
```

Comment:
I have missed this in the initial implementation somehow. Why is it needed to append `Human:`, etc?
I can imagine htis cna be useful for Llama2 and co, but not for all models.
I would at least make those prefixes empty by default and remove the space if prompt is empty.